msteams-platform/concepts/build-and-test/teams-developer-portal.md

```diff
@@ -9,6 +9,33 @@ ms.author: surbhigupta
 
 # Manage your Teams apps using Developer Portal
 
```

Comment:
t is very odd that the first section of the DevPortal doc has a half-baked reference to AppStudio. This is screaming the assumption that everyone who is using Developer Portal has to be coming from AppStudio. This is highly confusing. If anything, this table should have been on the AppStudio page to tell developers "Why they should migrate to Developer Portal"